pages/settings/[userId].js

```diff
@@ -66,106 +109,31 @@ const UserSettings = () => {
     }
   }, [user, canAccess, loading, error, currentUser])
 
-  // Loading
-  if (loading || !currentUser.finished || !canAccess || !user) {
-    return (
-      <Layout title="Profile">
-        <LoadingPage />
-      </Layout>
-    )
-  }
-
   return (
     <Layout title="User Settings" hasDivider={true}>
-      <section className="user-profile-settings-wr">
-        <div className="user-profile-view">
-          <div className="cover">
-            {user.profile.cover && <img src={user.profile.cover.path} />}
-          </div>
-          <div className="user-profile-view-content">
-            <div className="user-profile-box-wrapper">
-              <UserProfileBox user={user} />
+      {cantSeeContent ? (
```

Comment:
if it's possible dont use negative variables , better to use `canSeeContent = false`